create_graphics.py

```diff
@@ -426,6 +426,7 @@ def parallel_maps(cla, fhr, ds, level, model, spec, variable, workdir,
         airport_fn=AIRPORTS,
         ax=ax,
         grid_info=field.grid_info,
+        model_name=cla.model_name,
```

Comment:
The `cla.model_name` is not used in this fashion. It is meant to be whatever string you might like to put in your map title. Apologies that I didn't catch that in PR #103, but when we are requiring a specific string to enable a feature for a given model, we must be in control of that string. A user could run this with "Experiment 2 HRRR-HI" and it wouldn't work appropriately.

The way we've done it for other models is set the `model` key in the relevant image list, which ends up stored in `cla.images[0]`.